### ISSUE TITLE
Accessibility: Error and Warning highlights are too similar (especially on dark themes and for Colour-blind Users)

### ISSUE THREAD

**Issue by Chocodav3** [godotengine/godot#112387]
### Tested versions

4.5.1.stable (official)

### System information

Windows 11 (64-bit)

### Issue description

In the Godot Script Editor, error and warning highlights use very similar brown tones on dark backgrounds. For colour-blind users, these colours are nearly indistinguishable, meaning users like myself cannot reliably distinguish at a glance warnings from errors due to the similar hues and low contrast.

This affects Godot's Default theme, Grey theme, and to some extent even Follow System Theme for dark mode users.

<img width="860" height="286" alt="Image" src="https://github.com/user-attachments/assets/69de336a-ba01-4600-8132-c4b2f49ddf61" />

As shown, warnings and error highlights colour-pick to "brown".

Currently these colours are hard coded and not exposed to Editor Settings -> Interface -> Theme (or Accessibility)
Please consider adding either user definable hues, or more contrasting/brighter red and yellow highlights as an option.

### Steps to reproduce

1. Open any script in the Script Editor
2. Add a line like
```
func _process(delta):
    pass
```
(you’ll get a warning: “delta is never used in the current function”)
3. Add a syntax error, e.g. remove indentation or misspell func. 
4. Observe the line highlight colours for the warning and the error.

### Minimal reproduction project (MRP)

any script:
```
func this_is_an_error()

func this_is_a_warning(delta):
	pass
```

**Comment by aMadReason** [comment]
+1 for adding configurable colors for syntax highlighting in the editor. It would be really good for handling color contrast issues by allowing the user to set what's best for them.

**Comment by Giganzo** [comment]
If i am not misunderstanding there should already be settings for changing the color for error and warning. They are found in Editor Settings > Text Editor > Theme

Error: is called mark_color

Warning: warning_color

<img width="445" height="171" alt="Image" src="https://github.com/user-attachments/assets/ad25b89a-f176-4652-a711-e42fbc65337c" />

Maybe the default colors can be improved or/and make it easier to find these theme settings?
There is a Theme option under File in the script editor maybe an option to open the theme settings directly from there could be an option?

**Comment by Chocodav3** [comment]
Oh wow! Thank you so much!
Even after hunting through the documentation and spending an age looking through Interface Themes and Text Editor Appearance settings... I don't think I would have ever found those on my own.
It definitely would be nice to see user friendlier options appear in Accessibility one day though.
That's super appreciated, Giganzo.
